blobs/howto/python/blob-devguide-py/blob-devguide-blobs.py

```diff
@@ -0,0 +1,410 @@
+import io
+import os, uuid
+import random
+import time
+from azure.core.exceptions import HttpResponseError, ResourceExistsError
+from azure.identity import DefaultAzureCredential
+from azure.storage.blob import BlobServiceClient, BlobClient, ContainerClient, BlobLeaseClient, BlobPrefix, ContentSettings
+
+class BlobSamples(object):
+
+    # <Snippet_upload_blob_data>
+    def upload_blob_data(self, blob_service_client: BlobServiceClient, container_name):
+        blob_client = blob_service_client.get_blob_client(container=container_name, blob="sample-blob.txt")
+        data = "Sample data for blob"
+
+        # Upload the blob data - default blob type is BlockBlob
+        blob_client.upload_blob(data, blob_type="BlockBlob")
+    # </Snippet_upload_blob_data>
+
+    # <Snippet_upload_blob_stream>
+    def get_random_bytes(self, size):
```

Comment:
You could consider replacing this helper method with just `os.urandom(size)`. Just a simpler way to generate some random bytes. We use this in our tests quite frequently.
https://docs.python.org/3/library/os.html#os.urandom

There is also `random.randbytes(size)` but that was only introduced in Python 3.9 so not sure it'd be great to use in the samples since we support down to Python 3.7.